keadm/cmd/keadm/app/cmd/util/cloudinstaller.go

```diff
@@ -20,6 +20,7 @@ type KubeCloudInstTool struct {
 	Common
 	AdvertiseAddress string
 	DNSName          string
+	PackagePath    	 string
```

Comment:
Add comment to show what is packagePat6h

Reply:
i have given the explanation of PackagePath in keadm/cmd/keadm/app/cmd/common/constant.go , is that ok?

Reply:
@Rachel-Shao when you handle a comment and think it fixed, you can click "Resolve" button to finish it.